src/customizations/volto/helpers/Html/Html.jsx

```diff
@@ -88,18 +101,38 @@ class Html extends Component {
           {head.link.toComponent()}
           {head.script.toComponent()}
           <meta property="og:type" content="website" />
+
           <script
             dangerouslySetInnerHTML={{
-              __html: `window.env = ${serialize(runtimeConfig)};`,
+              __html: `window.env = ${serialize({
+                ...runtimeConfig,
+                // Seamless mode requirement, the client need to know where the API is located
+                // if not set in the API_PATH
+                ...(apiPath && {
+                  apiPath,
+                }),
+                ...(publicURL && {
+                  publicURL,
+                }),
+              })};`,
             }}
           />
           <link rel="shortcut icon" href="/favicon.ico" />
+          <link rel="icon" href="/favicon.ico" sizes="any" />
+          <link rel="icon" href="/icon.svg" type="image/svg+xml" />
+          <link
+            rel="apple-touch-icon"
+            sizes="180x180"
+            href="/apple-touch-icon.png"
+          />
+          <link rel="manifest" href="/site.webmanifest" />
```

Comment:
Questi li avevamo tolti perché spesso non li mettiamo nei siti (per negligenza) e danno errori in console che è meglio non avere. Sarebbe interessante tenerli ma dobbiamo mettere nella nostra guida all'upgrade dei default sani in modo da non avere errori. Altrimenti, per ora lascerei commentato.